nixos/tests/harmonia.nix

```diff
@@ -13,6 +13,9 @@
 
       networking.firewall.allowedTCPPorts = [ 5000 ];
       system.extraDependencies = [ pkgs.emptyFile ];
+
+      # check that extra-allowed-users is effective for harmonia
+      nix.settings.allowed-users = [];
```

Comment:
We should also try to download a store entry to make sure this works end to end.

Reply:
Is this not done in:

```
    client01.succeed("nix-store --realise ${pkgs.emptyFile} --store /root/other-store")
```
?

Reply:
true, wasn't obvious to me at first.